packages/commonwealth/server/passport/magic.ts

```diff
@@ -249,61 +299,21 @@ async function loginExistingMagicUser({
       await ssoToken.save({ transaction });
       log.trace('SSO TOKEN HANDLED NORMALLY');
     } else {
-      // situation for legacy SsoToken instances:
-      // - they only have profile_id set, no issuer or address_id
-      // we will locate an existing SsoToken by profile_id, and migrate it to use addresses instead.
-      // if none exists, we will create it
-      malformedSsoToken = await models.SsoToken.scope(
```

Comment:
Do we know how many existing users will be affected by this removal?

Reply:
profile_id was already removed from SsoTokens and all records have issuer data